usaspending_api/references/v2/views/filter_tree/psc_filter_tree.py

```diff
@@ -217,9 +217,8 @@ def _combine_nodes(self, upper_tier, lower_tier):
             for lower_node in lower_tier:
                 if upper_node["id"] in lower_node["ancestors"] and lower_node["id"] not in node_ids:
                     children.append(lower_node)
-            sorted(children, key=lambda x: x["id"])
```

Comment:
This line wasn't actually ordering `children`. It wasn't to root cause of the top-level PSC ordering issue but prompted the larger refactor